app/assets/javascripts/email-signup-analytics.js

```diff
@@ -0,0 +1,18 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function EmailSignupAnalytics () {}
+
+  EmailSignupAnalytics.prototype.trackSubscriptions = function (params) {
+    var $button = params.$button
+    var category = params.category
+    var action = params.action
+
+    $button.addEventListener('click', function (e) {
```

Comment:
Super picky - you don't need to pass `e` through if you're not going to use it:

```suggestion
    $button.addEventListener('click', function () {
```